libs/hdf-converters/src/snyk-mapper.ts

```diff
@@ -37,132 +40,580 @@ function nistTag(identifiers: unknown[]): string[] {
+                      isApi?.apiVersion === 1 ? SNYK_API_ISSUE_DATA_PATH : ''
+                    }description`,
+                    transformer: (description: string): string => {
+                      return (
+                        description
+                          .split(/(?:^|\s)## /g)
+                          .find((section) => section.startsWith('References'))
+                          ?.substring('References'.length)
+                          ?.trim() || ''
+                      );
+                    }
+                  }
+                }
+              ],
+              impact: {
+                path: `${
+                  isApi?.apiVersion === 1 ? SNYK_API_ISSUE_DATA_PATH : ''
+                }severity`,
+                transformer: impactMapping(IMPACT_MAPPING)
+              },
+              code: {
+                transformer: (item: Record<string, unknown>): string => {
+                  return JSON.stringify(item, null, 2);
+                }
+              },
+              results: [
+                {
+                  status: ExecJSON.ControlResultStatus.Failed,
+                  code_desc: {
+                    ...(!isApi && {path: 'from'}),
+                    transformer: (input: unknown): string => {
+                      let paths = input;
+                      if (isApi?.apiVersion === 1) {
+                        paths = Object.values(
+                          _.pick(
+                            _.get(this.unconvertedData, 'dependencyChains'),
+                            (_.get(input, 'pkgVersions') as string[]).map(
+                              (version) =>
+                                `${_.get(input, 'pkgName')}@${version}`
+                            )
+                          )
+                        );
+                      }
+                      if (Array.isArray(paths) && paths.length > 0) {
+                        if (!Array.isArray(paths[0])) {
+                          paths = [[paths]];
+                        } else if (!Array.isArray(paths[0][0])) {
+                          paths = [paths];
+                        }
+                        return (paths as string[][][]) // versions/multiple paths/specific path
+                          .map((path) =>
+                            path
+                              .map((p) => `From : [ ${p.join(' , ')} ]`)
+                              .join('\n')
+                          )
+                          .join('\n');
+                      } else {
+                        return '';
+                      }
+                    }
+                  },
+                  start_time: ''
+                }
+              ]
+            }
+          ],
+          sha256: ''
         }
+      ],
+      passthrough: {
+        transformer: (data: Record<string, unknown>): Record<string, unknown> =>
+          isApi?.apiVersion === 1
+            ? {
+                snyk_unmapped_data: _.omit(data, [
```

Comment:
Rename `snyk_unmapped_data` to `other_source_tool_data`.

Reply:
based off discussion from earlier today, i'm renaming it to that auxiliary data thing unless aaron prefers something else